apicast/src/configuration.lua

```diff
@@ -129,12 +128,34 @@ end
 
 local Service = require 'configuration.service'
 
+local noop = function() end
+local function readonly_table(table)
+    return setmetatable({}, { __newindex = noop, __index = table })
+end
+
+local empty_t = readonly_table()
+
+local function backend_endpoint(proxy)
```

Comment:
The indentation here is not consistent with the rest of the codebase.

Reply:
Indeed. Haven't configured my IDE and used the default 4 spaces indentation. 
Must admit that I quite like it. Somehow it makes it more readable. Also the OpenResty code is indented with 4 spaces. 
I'd consider switching to 4 spaces. Wdyt?

Reply:
I don't mind as long as it's spaces :) and we are consistent across the codebase.
http://lua-users.org/wiki/LuaStyleGuide